### ISSUE TITLE
Add Zimbabwe, Pakistan, Boliva, Malaysia, Kazakhstan and ANZ plan updates to chainvar

### ISSUE THREAD

**Issue by mikev** [helium/miner#1791]
Malaysia cannot use AS923_1 because it uses illegal frequencies above 924 MHz.  Recently the MCC announced that Helium would address the Malaysia frequency issues.

Work to create a new plan AS923_1B began in May and included engineering work on the Miner repo.  However testing from Malaysian engineers demonstrate that 'Miner Info Region' still returns AS923_1 within Malaysia.

This is a critical issue for Malaysian Hotspots and needs to be urgently addressed.

Engineering work for Malaysia was tracked with this parent ticket
https://github.com/helium/router/issues/664

The engineering work to add AS923_1B to Miner was done in this PR
https://github.com/helium/miner/pull/1664

**Comment by mikev** [comment]
MCC Announcement

In Malaysia the allowed frequency is 919-923MHz with an EIRP of 500 mW (max), and 923-924 MHz with an EIRP of 500 mW (max).  Frequencies at or above 924 Mhz are not allowed.
The official Malaysia MCMC document is here: 
Legally, Helium must follow the local Government regulations. In ~2 weeks, on Thursday, July 28th, Helium's compatible LoRaWAN frequency in Malaysia will be AS923-1.

AS923-1 Uplink channels:
        u_channels = [922.0, 922.2, 922.4, 922.6, 922.8, 923.0, 923.2, 923.4],

According to our ETL data, there are ~73 total online hotspots in the Malaysian region and ~1700 online and offline hotspots.
If you are a manufacturer who would like to sell into Malaysia, please be advised that you'll need to get Malaysian Communications and Multimedia Commission (MCMC) certified and share that certification with the Manufacturing Compliance Committee (MCC).
If you already have MCMC certification and have sold hotspots into the Malaysia region, please contact your customers and let them know that after July 28th, the hotspots will update to conform to Malaysia regulations.
Hotspots sold or active in the region require MCMC certification.
If you are a manufacturer supporting hotspots in the region and have questions or need support, please reach out.

**Comment by vihu** [comment]
We need to update the following set of chain variables as I understand it.

- `regulatory_regions` (this should be extended to add `region_as923_1b`
- `region_as923_1` needs to be changed and `region_as923_1b` needs to be added (according to https://github.com/helium/lorawan-h3/pull/12) (in serialized form, we have helpers in core to do this).
- `region_as923_1b_params` needs to be added (serialized form)

The existing regulatory region chain vars for reference are:
```erlang
(val_miner@127.0.0.1)5> rp(blockchain:config(regulatory_regions, blockchain:ledger())).         
{ok,<<"region_as923_1,region_as923_2,region_as923_3,region_as923_4,region_au915,region_cn470,region_eu433,region_eu868,region_in865,region_kr920,region_ru864,region_us915">>}
```

It would be good to collate these chain vars for inspection, I can try to get to this next week.
The only thing I'm unsure about is whether `region_as923_1_params` has changed or not. We can cross check the values on chain.

**Comment by mikev** [comment]
@vihu You wrote "region_as923_1 needs to be changed"

Why?  Nothing has changed for the existing region AS923_1.  It should remain unmodified I believe.

**Comment by vihu** [comment]
> @vihu You wrote "region_as923_1 needs to be changed"
> 
> Why? Nothing has changed for the existing region AS923_1. It should remain unmodified I believe.

I thought `AS923_1` got changed https://github.com/helium/lorawan-h3/pull/12 and https://github.com/helium/lorawan-h3/pull/12 according to the diff at least.

**Comment by mikev** [comment]
AS923_1B is not on the chainvar.  It needs to be set with the special Malaysia frequencies.

**Comment by oberchoo** [comment]
Looking forward to closing this issue so Helium will be in compliance with Malaysia operating freq.

**Comment by kiate** [comment]
Hi, thanks for your work @mikev & @vihu! Will this issue be resolved in any recent update? It's going to be a great news for fellow Malaysian!

**Comment by NorHairil** [comment]
Woah finally someone has come out and spoken about the frequency issue. We can finally mine HNT safely if this issue is solved :) 
Great work from you guys @mikev @vihu! Looking forward to the updates.

**Comment by mikev** [comment]
We'll add Zimbabwe, Pakistan, Boliva, Malaysia and ANZ plan to this ticket.  This includes mirroring the changes in https://github.com/helium/lorawan-h3/pull/12 into the chainvar.

**Comment by abhay** [comment]
Makes sense. What's your target for this change, @mikev @vihu @Vagabond? 1.15.x or 1.16.x? There are some things urgent in 1.15.x that need to go out quickly (next week most likely) and 1.16.x already has one thing we know about that'll have to go in (more likely a mid October release)

**Comment by mikev** [comment]
@abhay - if 1.16.x is scheduled for mid-October, then I'd prefer we put these into 1.15.x and release them next week.
We have multiple external parties waiting on these changes.

**Comment by mikev** [comment]
The h3 changes just merged.  We're ready to go!
https://github.com/helium/lorawan-h3/pull/16

**Comment by mikev** [comment]
Issue is dependent on automatic reboot upon region change:
https://github.com/helium/nextgate/pull/849

**Comment by oberchoo** [comment]
> Issue is dependent on automatic reboot upon region change: https://github.com/helium/nextgate/pull/849

It seems the repo is locked? Anyway, when will this dependent on automatic reboot upon region change be closed?

**Comment by mikev** [comment]
Product has planned the chain update for Nov 11th

**Comment by oberchoo** [comment]
Great! Looking forward, I assume this chain update will also do the automatic reboot upon region change. And the new freq plans are already in the chain variables, right?

**Comment by mikev** [comment]
Adding Kazakhstan update from EU433 to IN865

**Comment by oberchoo** [comment]
How can we know if the changes are being made? The region should be changed automatically and all the beacons and witnesses should shift to new freq plan automatically too?

**Comment by oberchoo** [comment]
Today is already the 12th of November, as far as I checked from https://explorer.helium.com/hotspots,  hotspots in Malaysia are still beaconing and witnessing the old freq plan, which exceeds 923.4 MHz. Is a manual reboot necessary for the new plan to be active?

**Comment by mikev** [comment]
Chainvar deployed on Nov 17th.
